### PR TITLE
perf(sample): add non-sso tab sample in sample gallery

### PR DESCRIPTION
**Pull request by KennethBWSong**
UI:
image

### CODE REVIEW

File: packages/fx-core/src/common/samples-config.json [hunk 1]
```diff
@@ -126,6 +126,16 @@
             "time": "45min to run",
             "configuration": "Manual configurations required",
             "suggested": false
+        },
+        {
+            "id": "hello-world-tab-without-sso",
+            "title": "Hello World Tab",
+            "shortDescription": "A simple Microsft Teams Tab application displays static content",
+            "fullDescription": "This is a simple static tab application with tutorial to include single sign on features.",
+            "tags": ["TAB", "TS"],
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/common/samples-config.json:135]: Align the wording as 'Tab'?

**Comment by HuihuiWu-Microsoft** [comment]
Please make sure responsive layout tested and the gif reviewed by @jaeyonglee8687

File: packages/fx-core/src/common/samples-config.json (hunk 1 above)
Comment (KennethBWSong) [reply packages/fx-core/src/common/samples-config.json]: Fixed. Thx!

**Comment by KennethBWSong** [comment]
> Please make sure responsive layout tested and the gif reviewed by @jaeyonglee8687

As shown in the description, the UI works as expected and Jaeyong have reviewed the gif. Thx!

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by MuyangAmigo — APPROVED** [review]
The wording part looks good to me.

**Pull request merged**